spec/services/list_languages_taught_in_modern_language_course_spec.rb

```diff
@@ -0,0 +1,20 @@
+describe ListLanguagesTaughtInModernLanguageCourse do
+  describe '#call' do
+    context 'when the course is for Modern Languages (Spanish with German)' do
```

Comment:
I'm unable to write test cases for other contexts as I'm not clear what courses are applicable and which aren't.